tcti/tcti_device.map

```diff
@@ -1,6 +1,8 @@
 {
     global:
         InitDeviceTcti;
```

Comment:
Should InitDeviceTcti be private now?

Reply:
Eventually I'm hoping that it'll go away completely. Removing the old API at the same time the new one is introduced seemed a bit drastic though. To make the transition the tools and tabrmd would need PRs to go in at the same time. Since we don't have those PRs ready and @williamcroberts is out of town I figured we could avoid breakage by keeping the old API in place for a bit, though it should be removed before the 2.0 release.

Reply:
That seems fine, one patch adding them in a backwards compatible way, one patch removing them. This pretty much lets me kill off the tcti init in the tools, as I provided a common interface, so I am glad to get rid of that.

Reply:
I'm thinking we'll have a TCTI utility library that provides proper functions for the TCTI function pointers (the macros we have now are good but they're hostile to language bindings) as well as the `dlopen` magic for loading. I'll pick this up after this PR is merged. Should probably get a ticket in the issue tracker too.